libraries/onos.robot

```diff
@@ -59,6 +59,8 @@ Execute ONOS CLI Command use single connection
     ...                In case no connection is open a connection will be opened
     ...                Using Write and Read instead of Execute Command to keep connection alive.
     [Arguments]    ${host}    ${port}    ${cmd}
+    Log To Console      \n'Execute ONOS CLI Command use single connection':
```

Comment:
A way to do it could be using the "Run Keyword If" builtin keyword to execute the "Log To Console" keyword only if an environment variable matches something specific. The environment variables can be read using the "Get Environment Variable" builtin keyword.

With that, we could implement a new custom keyword (i.e. in the voltha-system-tests/libraries/utils.robot file) that would execute first the keyword to read the env var, and after that the keyword with condition debugging/verbose output. Then, we would add a call to this custom keyword in whichever test we wanted to add debugging to.

Would you like those changes to be included in the PR or is it a thought for future modifications?

Reply:
This was mainly a thought for future enhancements.  If the custom keyword is easy to code feel free to add it in and start using it but def not a critical feature.